rali/rali/include/image_loader.h

```diff
@@ -54,6 +58,10 @@ class ImageLoader : public LoaderModule {
     LoaderModuleStatus load_routine();
     Image* _output_image;
     std::vector<std::string> _output_names;//!< image name/ids that are stores in the _output_image
+    std::vector<uint32_t> _original_width;
```

Comment:
looks like those are part _decoded_image_info. Please use that instead and remove duplicate member variables